app/tests/unit/components/Form/ProjectReportingRequirementsForm.test.tsx

```diff
@@ -0,0 +1,266 @@
+import { screen, act, within, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import ProjectQuarterlyReportForm from "components/Form/ProjectQuarterlyReportForm";
+import { graphql } from "react-relay";
+import ComponentTestingHelper from "tests/helpers/componentTestingHelper";
+import compiledProjectQuarterlyReportFormQuery, {
+  quarterlyReportsFormQuery,
+} from "__generated__/quarterlyReportsFormQuery.graphql";
+import { ProjectQuarterlyReportForm_projectRevision } from "__generated__/ProjectQuarterlyReportForm_projectRevision.graphql";
+
+const testQuery = graphql`
+  query ProjectReportingRequirementsFormQuery @relay_test_operation {
+    query {
+      # Spread the fragment you want to test here
+      projectRevision(id: "Test Project Revision ID") {
```

Comment:
Using "Test Project Revision ID" seems to work fine, but I don't understand its relationship to `defaultMockResolver` below